src/hotspot/share/gc/z/zHeap.cpp

```diff
@@ -435,7 +435,7 @@ bool ZHeap::print_location(outputStream* st, zpointer ptr) const {
   const zaddress addr =  ZPointer::uncolor(ptr);
 
   if (addr == zaddress::null) {
-    st->print_raw_cr("NULL");
+    st->print_raw_cr("nullptr");
```

Comment:
I'd prefer if this were left as either NULL or null.

Reply:
Suggest: null